example/test/MutableReferenceSpec.hs

```diff
@@ -115,13 +116,13 @@ prop_shrinkForkMinimal = shrinkPropertyHelper (prop_parallel RaceCondition) $ \o
     = anyRose isMinimal
     . rose (liftShrinkFork shrink1)
     where
-    anyRose :: (a -> Bool) -> Rose a -> Bool
-    anyRose p (Rose x xs) = p x || any (anyRose p) xs
+    anyRose :: (a -> Bool) -> Tree a -> Bool
+    anyRose p (Node x xs) = p x || any (anyRose p) xs
```

Comment:
`foldTree` isn't part of `Data.Tree` in the `containers` version we are using... 

(Also an unfold followed by a fold is suggest we really need `hyloTree` :-).